g3doc/user_guide/compatibility.md

```diff
@@ -42,6 +42,9 @@ Most common utilities work. Note that:
 
 *   Some tools, such as `tcpdump` and old versions of `ping`, require explicitly
     enabling raw sockets via the unsafe `--net-raw` runsc flag.
+    * In case of tcpdump the following invocations will work
+        * tcpdump -i any
+        * tcpdump -i <device-name> -p  (-p is required to disable promiscuous mode)        
```

Comment:
Optional: both `tcpdump` commands could be surrounded by backticks for code font, e.g. \`tcpdump -i any\`